scripts/md2html/build-src.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Author: @ralfhandl
+
+# run this script from the root of the repo
+
+mkdir -p deploy/oas
+mkdir -p deploy/js
+mkdir -p deploy-preview
+
+cp -p node_modules/respec/builds/respec-w3c.* deploy/js/
+
+cd scripts/md2html
+
+filename=src/oas.md
+tempfile=../../deploy/oas/oas.html
```

Comment:
it seems odd to me that we're putting a tempfile under /deploy when our output is under /deploy-preview.  Should we not have the tempfile there and named something obvious (eg. with `tmp` or `temp` in the name?) so that if something breaks the mess is left in the same place we were expecting output?  And we can have an `npm run clean-src` that just deletes that directory so that it works whether the build was successful or not? (I don't care if you add clean-src in this PR or not, I'm just thinking it might be nice at some point).